### ISSUE TITLE
Suggestion modal

### ISSUE THREAD

**Issue by LibbyUX** [hubmapconsortium/hra-ui#387]
![image](https://github.com/hubmapconsortium/hra-ui/assets/97060856/e758da8c-1c2e-42e3-9a24-fc86381097cb)

**Comment by LibbyUX** [comment]
Conversations began occurring on #508 about the placement of contact buttons around our current production interfaces.

Due to scope creep, Ushma and I decided to keep things simple for our initial interfaces. Currently, this email contact button is located in our footers (see [nav footer](https://www.figma.com/design/gQEMLugLjweDvbsNNUVffD/Design-System-Repository?node-id=2863-4503&t=7cuR5QQ0OcN2tpLA-4) in Figma Design System Repo), however, we do want to achieve a [floating action button](https://m3.material.io/components/floating-action-button/overview), eventually, for a contact/suggestion flow for users.

I am mentioning @bherr2 & @emquardokus to bring this conversation here. Thank you all!

**Comment by emquardokus** [comment]
User interface tools should be harmonized in style.
1) ASCT+B explorer has different footer than others https://hubmapconsortium.github.io/ccf-asct-reporter/. Contact in footer.

2) the RUI doesn't have a contact button or information https://apps.humanatlas.io/rui/. In the "i" to access information, contact should exist. Also the funding in acknowledgment should be updated.

3) the EUI is similar set up to RUI in that no contact from within. An "i" button leads to information but it's old and references unpublished work that is now published.

4) The functional tissue unit explorer https://apps.humanatlas.io/ftu-explorer/#/ has a contact button in the upper right of window going to cnscntr@indiana.edu rather than the infoccf@indiana.edu which is used for the footer "contact HRA team". This should be changed to use infoccf@indiana.edu.
In addition, as per GitHub issue #508, all indiana.edu should be changed to iu.edu.

**Comment by bherr2** [comment]
Yep @LibbyUX is soon to start on updating _all_ HRA User Interfaces to use the HRA Design System and should be able to address this common feature to all our interfaces. Thanks for spelling these out!
